lib/collections/phoneNumber.js

```diff
@@ -0,0 +1,87 @@
+phoneNumber = new Mongo.Collection('sites1');
```

Comment:
@ahmedraza11 hey, `sites1` is already used in sites. You have to put another name for phoneNumbers and for Assets.
And It should be like this
`PhoneNumber = new Mongo.Collection('phoneNumber');`